tests/integration/cartography/intel/lastpass/test_users.py

```diff
@@ -0,0 +1,38 @@
+import cartography.intel.lastpass.users
+import tests.data.lastpass.users
+
+
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_lastpass_users(neo4j_session):
```

Comment:
also add a test for humans and tenants

Reply:
@resilience-jychp - can you let `sync()` be the function under test and patch out the results of `get()` with `tests.data.lastpass.users.LASTPASS_USERS`? 

This way as the Lastpass module grows you don't need to add more transform + load calls and you can just add more test cases.

Here's an example of doing this: https://github.com/lyft/cartography/blob/ab1584f325c8a9c424c1358f2e482b9c4df2c7ea/tests/integration/cartography/intel/aws/ec2/test_ec2_instances.py#L20-L37